src/_glass.scss

```diff
@@ -0,0 +1,13 @@
+// "glass" style mixin
+@mixin glass-outer {
+  border: 1px solid rgba(255, 255, 255, 0.25); // stroke should be applied to the outer div
```

Comment:
This doesn't look like the right gradient. I've been playing with it and it doesn't seem like it's possible to do what the design asks for. I'm contacting Eth to see what he'd like to do.